app/src/main/java/com/jienihou/outfitxchange/models/CategoryItem.kt

```diff
@@ -0,0 +1,7 @@
+package com.jienihou.outfitxchange.models
+
+data class CategoryItem (var imageName: Int, var category: String){
+    override fun toString(): String{
```

Comment:
Do we need this? Data classes usually have an [automatic toString method](https://kotlinlang.org/docs/reference/data-classes.html)?